taxi/models.py

```diff
@@ -0,0 +1,30 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=255, unique=True)
+
+    class Meta:
+        verbose_name = "Driver"
+        verbose_name_plural = "Drivers"
+
+    def __str__(self):
+        return f"{self.first_name} {self.last_name}"
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+    def __str__(self):
+        return f"{self.name} {self.country}"
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=255)
+    manufacturer = models.ForeignKey(Manufacturer, on_delete=models.CASCADE)
+    drivers = models.ManyToManyField(Driver)
```

Comment:
u should use AUTH_USER_MODEL) 
dont forget read [checklist.md](https://github.com/mate-academy/py-taxi-service-startproject/blob/master/checklist.md)

Reply:
also add related_name for Car model